### PR TITLE
update 01-retitle.Rmd

### PR DESCRIPTION
**Pull request by MarkLaVenia**

### CODE REVIEW

File: R/dsieur_ch5n6_script.R [hunk 1]
```diff
@@ -58,3 +58,5 @@ ma_data_init %>%
   count() %>% 
   filter(n > 10) %>% 
   arrange()
+
+yxwexnwe
```
Comment (tanho63) [inline R/dsieur_ch5n6_script.R]: Hi Mark - is this last bit intentional

Comment (MarkLaVenia) [reply R/dsieur_ch5n6_script.R]: No, not at all. I'm sorry. I have no idea how that got in there. Thanks for catching!

Comment (MarkLaVenia) [reply R/dsieur_ch5n6_script.R]: Oh wait, yeah that was in the script file. Yes, pls add if it is no big deal. Thanks!!!

Comment (ivelasq) [reply R/dsieur_ch5n6_script.R]: Hi all! Mark, is this to show how to debug?

**Comment by MarkLaVenia** [comment]
Hmm? Maybe I'm not sure. Just reject it. Sorry. M

File: R/dsieur_ch5n6_script.R (hunk 1 above)
Comment (tanho63) [reply R/dsieur_ch5n6_script.R]:
```suggestion

```

**Review by tanho63 — APPROVED** [review]

**Pull request merged**